### PR TITLE
Update each integrations page to include beacon and trip attributes

### PR DESCRIPTION
**Pull request by bschoeneweis**
## What?
- Updating the user/event data sections for each integrations page to include beacon and trip attributes
- Change type definitions in user sections for arrays to be consistent

### CODE REVIEW

File: docs/integrations/amplitude.mdx [hunk 1]
```diff
@@ -546,3 +555,50 @@ Radar Event Field | Amplitude Event Property | Type | Example Value
...
+--- | --- | --- | ---
+`beacon._id` | `Beacon ID` | string | `"5b2c0906f5874b001aecfd8f"`
+`beacon.description` | `Beacon Description` | string | `"Store #123 - Drive-Thru"`
+`beacon.tag` | `Beacon Tag` | string | `"drive-thru"`
+`beacon.externalId` | `Beacon External ID` | string | `"123"`
+`beacon.metadata[{key}]` | `Beacon Metadata {key}` | {type} | `{value}`
+`confidence` | `Confidence` | string | `"high"`
+`duration` | `Duration` | number (minutes) | `1.42`
+
+If [Regions](/regions) is enabled, Radar will also send the following attributes:
```
Comment (jsani-radar) [inline docs/integrations/amplitude.mdx:594]: this part seems to be present on this integration only rather than across all of them, is this intended?

**Review by jsani-radar — COMMENTED** [review]
small comment - looking good otherwise

File: docs/integrations/amplitude.mdx (hunk 1 above)
Comment (bschoeneweis) [reply docs/integrations/amplitude.mdx:594]: @jsani-radar not sure on this one.  I saw it with the other event types and so I copied the pattern for this page.  Any input here @brettguenther?

**Review by jsani-radar — APPROVED** [review]
lgtm!

**Pull request merged**